timeseries/src/autogluon/timeseries/models/local/statsforecast.py

```diff
@@ -245,6 +245,17 @@ def _get_model_type(self):
 
         return AutoETS
 
+    def _predict_with_local_model(
```

Comment:
where is this used? If I understand correctly this is a new functionality, but it is unclear where this is used.

Reply:
This method is used [here](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/local/abstract_local_model.py#L165) - the method generates forecast for a single time series using the local model.

With this change, we first check if time series is long enough for the provided `season_length`. Then we call the [method](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/local/statsforecast.py#L24) that is shared by all models from StatsForecast.

Reply:
Thank you!